content/404.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html prefix="og: http://ogp.me/ns#" lang="en"><script type="text/javascript">window["_gaUserPrefs"] = { ioo : function() { return true; } }</script><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+<html>
+	<head>
+		<title>Oops! 404 Not Found</title>
+		<meta name="robots" content="noindex">
+		<meta charset="utf-8" />
+		<meta http-equiv="refresh" content="10; url=/" />
+	</head>
+	<body class="ain">
+		<div class="main">
+			<h1>Oops! Page not found.</h1>
+			Will redirect your browser to the main page in 10 seconds.
+			<div class="logo">
+				<img src="/images/logo.svg">
+			</div>
+		</div>
+	</body>
+</html>
+
+<style>
```

Comment:
We should use Tailwind for this one to keep the CSS of the site in the same place and easier to maintain.

Reply:
Will check how can I do.

By the way, I tried to just use the same layouts or at least all includes, but then it seems like the page needs to be a md. When I tested, hugo didn't parse html. (but just serve it as is.) However, I don't know how I can add page specific headers(for meta) or js since the 404 page could be better when it redirects users to home or somewhere.

Any hints?

Reply:
I'm not totally sure about that 404 redirect but I like the idea of having something better than our current 404 page (nginx).

Reply:
pushed another commit to replace them with tailwindcss.